src/box/lua/init.c

```diff
@@ -70,40 +70,29 @@
 #include "box/lua/watcher.h"
 #include "box/lua/iproto.h"
 
-#include "mpstream/mpstream.h"
-
-static uint32_t CTID_STRUCT_TXN_SAVEPOINT_PTR = 0;
-
-#if ENABLE_SPACE_UPGRADE
-void
-box_lua_space_upgrade_init(struct lua_State *L);
-#endif
-
 #if ENABLE_AUDIT_LOG
-void
-box_lua_audit_init(struct lua_State *L);
+#include "lua/audit_log_impl.h"
```

Comment:
> Why `_impl`?
because you asked about it verbally when we discussed implementation, and README says about it as well: 
https://github.com/tarantool/tarantool-ee/#enterprise-implementation

Am I misinterpret this?

Reply:
`FOO_impl.h` suffix is required because there's `FOO.h` in the CE repository. This isn't true in your case - the header exists only in EE hence we don't need to bother with suffixes.

Reply:
Discussed verbally. Agreed to add `FOO.h` to the CE repo for each `FOO_impl.h` in the EE repo, like we do for EE headers in other directories.

Reply:
Did it so.